### PR TITLE
Rebuild harfbuzz4 and add izahn as maintainer

### PR DESCRIPTION
**Pull request by izahn**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/openjdk-feedstock/pull/90

### CODE REVIEW

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
The OSX build fails with 
```
+ codesign --remove-signature bin/jaotc
--sign is required
```
any ideas about how to fix that @conda-forge/openjdk ?

**Comment by izahn** [comment]
The `linux_aarch64` and `linux_ppcle64` builds both have this problem in the testing step:
```
qemu: fatal: Tried to call a TRAP
```
There is an open bug report about this at https://gitlab.com/qemu-project/qemu/-/issues/588

Options here include a) disable the test for `linux_aarch64` and `linux_ppcle64` or b) wait until an updated `qemu` is released with a fix for this issue. How should we proceed @conda-forge/openjdk ?

File: conda-forge.yml [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+  osx_arm64: osx_64
 conda_forge_output_validation: true
-provider: {linux_aarch64: default, linux_ppc64le: azure, win: azure}
-test_on_native_only: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: azure
```
Comment (hmaarrfk) [inline conda-forge.yml:11]:
```suggestion
  linux_aarch64: default
  linux_ppc64le: default
```

and rerender. This should make them both compile through travis. Hopefully qemu won't be a problem there. You have a 1 hour time limit. So maybe you'll get in right under it?

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
Tried Travis but got

```
The job exceeded the maximum log length, and has been terminated.
```

**Comment by isuruf** [comment]
What's the reason for the log being that large?

**Comment by hmaarrfk** [comment]
gcc seems to be pinned in ppc64le try to unpin in?

Also it seems that there is a likely bug with `register char c`

**Comment by hmaarrfk** [comment]
You can likely delete the file: https://github.com/izahn/openjdk-feedstock/blob/rebuild-harfbuzz4/recipe/conda_build_config.yaml

**Comment by izahn** [comment]
> What's the reason for the log being that large?

Log is at https://app.travis-ci.com/github/conda-forge/openjdk-feedstock/jobs/568466954

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Well if "just" ppc64le is failing on azure, you can likely skip the compilation test for that one while qemu is fixed.

It is really curious as to why the logs are so verbose on ppc64le

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
`linux_aarch64` fails with 

```
2022-04-28T11:42:38.3382062Z /home/conda/feedstock_root/build_artifacts/openjdk_1651143572459/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/10.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: skipping incompatible /home/conda/feedstock_root/build_artifacts/openjdk_1651143572459/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/10.3.0/../../../libasound.so when searching for -lasound
2022-04-28T11:42:38.3384235Z collect2: error: ld returned 1 exit status
2022-04-28T11:42:38.3434461Z make[3]: *** [Lib-java.desktop.gmk:64: /home/conda/feedstock_root/build_artifacts/openjdk_1651143572459/work/src/build-build/support/modules_libs/java.desktop/libjsound.so] Error 1
```
Anyone have ideas about this one?

**Comment by hmaarrfk** [comment]
sorry :(

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
> Well if "just" ppc64le is failing on azure, you can likely skip the compilation test for that one while qemu is fixed.

Ugh, so I tried this in https://github.com/conda-forge/openjdk-feedstock/pull/93/commits/4f6e23c3fc6d42b8d7392f4593b0de692af3ac71 https://github.com/conda-forge/openjdk-feedstock/pull/93/commits/f084698369942311309f5389091297a99a831fe7 but it looks like it still tries to run the test. What selector can we use to skip that test on both windows and ppc64le?

**Comment by hmaarrfk** [comment]
I think additve selectors were broken.

**Comment by hmaarrfk** [comment]
https://github.com/conda-forge/conda-smithy/issues/1617

**Comment by izahn** [comment]
> [conda-forge/conda-smithy#1617](https://github.com/conda-forge/conda-smithy/issues/1617)

Ah, thanks, I was starting to think my mind was going...

**Comment by izahn** [comment]
Still having problems with the aarch build. I tried setting `--with-alsa` to `PREFIX` / `BUILD_PREFIX` in https://github.com/conda-forge/openjdk-feedstock/pull/93/commits/e6ea9a0977be49405f990ab0f5f83c5ce3119f3d and https://github.com/conda-forge/openjdk-feedstock/pull/93/commits/f4d1fd45c1027a7bcf477c874071486011454e7c but either way we get
```
/home/conda/feedstock_root/build_artifacts/openjdk_1651249498229/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/10.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: skipping incompatible /home/conda/feedstock_root/build_artifacts/openjdk_1651249498229/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/10.3.0/../../../libasound.so when searching for -lasound
collect2: error: ld returned 1 exit status
make[3]: *** [Lib-java.desktop.gmk:64: /home/conda/feedstock_root/build_artifacts/openjdk_1651249498229/work/src/build-build/support/modules_libs/java.desktop/libjsound.so] Error 1
```
Any ideas? @xhochy I think you were the one that got a `aarch64` build working in the first place, any thoughts about this?

**Comment by izahn** [comment]
I can't figure out the `aarch64` alsa problem, if no one else has time to fix it I recommend merging this as-is and addressing the `aarch64` bug later @conda-forge/openjdk

**Comment by isuruf** [comment]
You are hitting https://github.com/conda/conda-build/issues/3155. A rebuild of `alsa-lib-feedstock` should fix this.

**Comment by hmaarrfk** [comment]
Seems like you might just be able to copy the migrator in. As yours might be the only feedstock that needs rebuilding

image

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
@conda-forge-admin please restart ci

**Comment by izahn** [comment]
@conda-forge-admin please restart ci

**Comment by izahn** [comment]
@conda-forge-admin please restart ci

**Comment by izahn** [comment]
Thanks to all your help @isuruf and @hmaarrfk I think this is ready.

**Comment by hmaarrfk** [comment]
Did you want to add yourself as a maintainer?

**Comment by izahn** [comment]
@conda-forge-admin please rerender

**Comment by izahn** [comment]
> Did you want to add yourself as a maintainer?

Sure, happy to help!

**Pull request merged**